elk-setting/docker-compose.yml

```
@@ -0,0 +1,208 @@
+volumes:  # 서비스들이 사용하는 데이터 저장소 설정
+  certs:  # 인증서 저장용 볼륨
+    driver: local  # 로컬 드라이버 사용
+  esdata01:  # Elasticsearch 데이터 저장용 볼륨
+    driver: local  # 로컬 드라이버 사용
+  kibanadata:  # Kibana 데이터 저장용 볼륨
+    driver: local  # 로컬 드라이버 사용
+  logstashdata01: # logstash 데이터 저장용 볼륨
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)